pkg/util/mount/mount.go

```diff
@@ -88,8 +88,10 @@ type HostUtils interface {
 	// GetFileType checks for file/directory/socket/block/character devices.
 	GetFileType(pathname string) (FileType, error)
 	// MakeFile creates an empty file.
+	// If pathname already exists, whether a file or directory, no error is returned.
```

Comment:
This isn't a comment that needs to be addressed in this patch, but is about the methods in general. If we are making a file or a directory, and a directory or file is there respectively, this would violate the principle of least surprise. I'm glad this patch adds clarification to the documentation to reduce the surprise potential.